src/gui/qgsmaptoolcapture.h

```diff
@@ -82,6 +82,11 @@ class GUI_EXPORT QgsMapToolCapture : public QgsMapToolAdvancedDigitizing
      */
     void deleteTempRubberBand();
 
+    /**
+     * Return defalut Z value
+     */
+    double getDefaultZValue() {return mDefaultZValue;};
```

Comment:
QGIS convention is not to name getters with a "get" prefix. This should be:

    double defaultZValue() const;